tox.ini

```diff
@@ -121,6 +121,39 @@ commands=
 commands_post=
     core: go tool covdata textfmt -i=.coverage -o coverage.txt
 
+# Add back mlflow in the mlflow PR
+[testenv:importer-wandb-py{38,39,310,311,312}]
```

Comment:
How about this?

It keeps config all in tox:
https://github.com/wandb/wandb/pull/6897/commits/6291cb871a886852b0cf6b73805207dd757a752b#diff-ef2cef9f88b4fe09ca3082140e67f5ad34fb65fb6e228f119d3812261ae51449L90-L155

Reply:
Does this part work?

```ini
commands=
    pytest {env:CI_PYTEST_SPLIT_ARGS:} -n=8 --durations=20 --reruns 2 --reruns-delay 1 --junitxml=test-results/junit.xml --cov --cov-report=xml --no-cov-on-fail --timeout 300 {posargs}
    importer-wandb: --wandb-second-server="true"
```

To me it looks like it would try running `--wandb-second-server="true"` as a command on its own. But if it works, that's very neat!